src/app/shared/register-github-app/register-github-app.component.html

```diff
@@ -0,0 +1,38 @@
+<div>
+  <p>
+    Install the Dockstore GitHub app on your repositories. This gives GitHub permission to notify Dockstore when pushes are made to GitHub.
+    For more details, see our
+    <a href="{{ Dockstore.DOCUMENTATION_URL }}/getting-started/github-apps/github-apps.html" target="_blank" rel="noopener noreferrer">
+      documentation <mat-icon>open_in_new</mat-icon> </a
+    >.
+  </p>
+  <p>Perform the following steps in order:</p>
+  <ol>
+    <li>Click the button below to install the Dockstore GitHub app on your GitHub organizations or specific repositories.</li>
+    <li>
+      Push a valid .dockstore.yml to the root of a GitHub repo that has the Dockstore GitHub app installed. The .dockstore.yml tells
+      Dockstore where the {{ entryType }}'s files are located.
+    </li>
+    <li>
+      It takes about 2 minutes after a push to GitHub for Dockstore to get notified and process the update. After those 2 minutes, refresh
+      your dashboard, and you should see the {{ entryType }}.
+    </li>
+    <li>
+      If you don't see the {{ entryType }}, navigate to the {{ entryType }}s section of My Dashboard, and click the Apps Logs link for the
```

Comment:
"navigate to the {{ entryType }}s section of My Dashboard" is a little confusing for me. Suggesting My <Entry>

Reply:
The problem is that I was concerned with is that the "tile" in the Dashboard is just labelled `Workflows`. The only place where `my-workflows` appears is in the url; the breadcrumb says "My Dockstore: Workflows".

But in writing this, I notice it says "My Dockstore", not "My Dashboard", so...